app/views/users/registration.html.erb

```diff
@@ -85,30 +78,40 @@
               </div>
             </div>
 
-            <br>
-            <div class="field">
-              <div class="control">
-                <label class="checkbox">
-                  <%= f.check_box :quality_acceptance, {:class => 'check-extra', :disabled => 'disable'} %>
-                  I have acknowledged these resources and 
-                  understand what a quality contribution is to open source.
-                </label>
+            <div class="checkboxes">
+              <% if @current_user.errors.messages[:quality_acceptance].include?("must be accepted") %>
```

Comment:
So one thing about moving these is that, if I don't make any mistakes on the first part of the form, I don't know what I did wrong until the second part, which might be confusing (since the error message will now only display on part two of the form).

Reply:
After chatting w/ Raven we agreed it'd be better to have the errors showing next to their relevant inputs, to avoid confusing. We'd had a few support tickets etc. as well from folks who'd missed the error messages because they weren't showing next to their respective inputs.

Reply:
Ok cool – whatever will be the least confusing! When I initially read through the change it made a lot of sense. Then just running through it, it seemed like it could be confusing to have to click through to the second part to read the error message(s) there. But if the feedback is that this would be less confusing that the current setup, then it definitely seems like the way to go.